site/pages/add-consequence.tsx

```diff
@@ -0,0 +1,144 @@
+import { NextPageContext } from "next";
+import { destroyCookie, parseCookies } from "nookies";
+import { ReactElement } from "react";
+import Radios from "../components/form/Radios";
+import { TwoThirdsLayout } from "../components/layout/Layout";
+import { COOKIES_ADD_CONSEQUENCE_ERRORS, COOKIES_ADD_CONSEQUENCE_INFO } from "../constants/index";
+import { InputInfo, ErrorInfo } from "../interfaces/index";
+
+interface AddConsequenceWithErrors {
+    errors: ErrorInfo[];
+    inputs: AddConsequenceProps;
+}
+
+export interface AddConsequenceProps {
```

Comment:
Can we move both of these to the interfaces file? We will need to import them elsewhere I assume

Reply:
done